Sources/LocalReceiptParsing/BasicTypes/ASN1Container.swift

```diff
@@ -64,6 +64,14 @@ struct ASN1Length: Equatable {
 
     let value: Int
     let bytesUsedForLength: Int
+    let definition: LengthDefinition
+
+    enum LengthDefinition: Int {
+
+        case definite
+        case indefinite
+
+    }
```

Comment:
this isn't really used, but it's useful when debugging

Reply:
I like it